CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# Contributing to Gitcoin
+
+Contributions to gitcoin could come in different forms. Some contribute code
+changes, others contribute docs, others help answer questions from users, help
+keep the infrastructure running,
+
+We welcome all contributions from folks who are willing to work in good faith
+with the community. No contribution is too small and all contributions are
+valued.
+
+* [Code of Conduct](#code-of-conduct)
+* [Issues](#issues)
+* [Discussions And General Help](#discussions-and-general-help)
+* [Pull Requests](#pull-requests)
+  * [Step 1: Fork](#step-1-fork)
+  * [Step 2: Branch](#step-2-branch)
+  * [Step 3: Code](#step-3-code)
+  * [Step 4: Commit](#step-4-commit)
+  * [Step 5: Rebase](#step-5-rebase)
+
+## Code of Conduct
+Contributions to Gitcoin are governed by the [Contributor Covenant version 1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html).
+All contributors and participants agree to abide by its terms. To report
+violations, shoot out an email to founders@gitcoin.co
+
+The Code of Conduct is designed and intended, above all else, to help establish
+a culture within the project that allows anyone and everyone who wants to
+contribute to feel safe doing so.
+
+Open, diverse, and inclusive communities live and die on the basis of trust.
+Contributors can disagree with one another so long as they are done in good
+faith and everyone is working towards a common goal.
+
+## Issues
+Issues in `gitcoin/web` are the primary means by which bug reports and
+general discussions are made. An contributor is allowed to create an issue,
+discuss and provide a fix if needed.
+
+## Discussions And General Help
+As Gitcoin is still at it's early stages, drop by gitcoincommunity.slack.com
```

Comment:
gitcoin.co/slack is where one can request an invite.. probably better to send folks here.